Gems/Atom/Feature/Common/Code/Source/FrameCaptureSystemComponent.cpp

```diff
@@ -497,7 +483,7 @@ namespace AZ
                 return InvalidFrameCaptureId;
             }
 
-            bool result = previewPass->ReadbackOutput(capture->m_readback);
+            bool result = previewPass->ReadbackOutput(captureState->m_readback);
```

Comment:
This looks to be the only place that uses the capture state in this function

Reply:
There are 3 places using this actually.
One for the swapchain readback and the other 2 for pass attachment readback.
`pass->ReadbackSwapChain(captureState->m_readback);`

Previewpass can be treated as a special pass attachment capture. I am also looking into have them more generalized so that it can be simplified further.